### DISCUSSION TITLE
Kiosk mode

### DISCUSSION THREAD

**Discussion by S1M8N** [matt8707/hass-config#75] [Q&A] [answered]
Hello !

First, thank you for your Dashboard, you are the best !

I just have a question :

Where is the alternative of "Kiosk mode" in your code ?

Right now, I have that in my ui-lovelace.yaml :

`kiosk_mode:
  hide_header: true`

Thank you for your next reply

Simon

**Comment by matt8707** [comment, marked as answer]
Hi, I'm using the query string option, only on tablet
[https://github.com/NemesisRE/kiosk-mode#query-strings](https://github.com/NemesisRE/kiosk-mode#query-strings)

**Reply by S1M8N** [reply]
Nice ! Thank you for your help !
